plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -297,7 +303,7 @@ public void testNonExistentWithoutIgnoreMissing() throws Exception {
 
     public void testIndexedChars() throws Exception {
         processor = new AttachmentProcessor(randomAlphaOfLength(10), null, "source_field",
-            "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 19, false, null);
+            "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 19, false, null, "resource_name");
```

Comment:
Let's not change the behavior of these tests:
```suggestion
            "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 19, false, null, null);
```